lib/game.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+require_relative 'ui.rb'
+
+class Game
+  attr_accessor :continue
+  def initialize(board, player1, player2, ui)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+    @ui = ui
+    @ui.show_board(@new_board)
+    play
+  end
+
+  def play
+    @player1.assign_sign(@ui)
+    @player2.sign = @player1.sign == 'X' ? 'O' : 'X'
```

Comment:
For instance, instead of assigning Player sign here, why don't we just initialize Player already with a sign? See my comment in `player.rb`.